hazelcast/src/main/java/com/hazelcast/internal/networking/nio/NioNetworking.java

```diff
@@ -103,6 +105,7 @@
     public NioNetworking(Context ctx) {
         this.threadNamePrefix = ctx.threadNamePrefix;
         this.metricsRegistry = ctx.metricsRegistry;
+        this.metricsRegistryScheduled = new AtomicBoolean(false);
```

Comment:
I would move this directly to the variable declaration to make the constructor less complex.

Reply:
done.